cli/cli.go

```diff
@@ -103,6 +105,13 @@ func NewApp(version, environment, hash, daemonURL, salt string,
 	cli.AppHelpTemplate = AppHelpTemplate
 	cli.SubcommandHelpTemplate = SubcommandHelpTemplate
 	cli.CommandHelpTemplate = CommandHelpTemplate
+	// Configure line wrapping for command descriptions
```

Comment:
Additional fix to make line wrapping tidier. Based on https://github.com/urfave/cli/pull/1119

Reply:
Correct line wrapping works if terminal is narrow. But if terminal is wide and only after `nordvpn mesh help` is shrunk - then lines are wrapped ugly like before. But this is unavoidable because of terminal nature.